src/cljs/harja/tiedot/urakka/laadunseuranta/laatupoikkeamat.cljs

```diff
@@ -105,3 +86,71 @@
                :loppuosa      losa
                :loppuetaisyys let})
         tiedot))))
+
+(defn- hoitokauden-alkuvuosi-nykyhetkesta [nyt]
+  (pvm/hoitokauden-alkuvuosi (pvm/vuosi nyt) (pvm/kuukausi nyt)))
+
+(defonce aikavali-atom (atom nil))
+(defonce tuck-tila (atom {:listaus-tyyppi :kaikki
```

Comment:
Tuck-tila tallennetaan yleensä harja.tiedot.urakka.urakka-namespaceen. Onko joku erityinen syy, miksi tämä on eri tavalla?

Reply:
Näin jälkiviisaana voi sanoa, että ei. Se oli loogista jossain vaiheessa muutosta, kun pyrin pitämään muutokset minimissä, mutta nyt tuo koko homma on jumpattu eri muotoon, niin olisi hyvä tämäkin siirtää oikeaan paikkaan. 
Eli hyvä pointti 👍 
Siirrän.